src/App.js

```diff
@@ -12,6 +20,10 @@ function App() {
   const { auth } = state;
   const token = auth.token;
 
+  const PrivateRoute = ({ children }) => {
+    return token ? children : <Navigate to="/" replace />;
```

Comment:
Maybe you could use the location's state here & navigate the user back to where he/she came from.

Reply:
Ok